frontend/apps/filemanager/filemanager.lua

```diff
@@ -470,6 +479,35 @@ function FileManager:onSwipeFM(ges)
     return true
 end
 
+function FileManager:addFileDialogButtons(row_id, row_func) -- FileManager, History, Collections file_dialog
```

Comment:
(Should this one be called addFileDialogButtonRow() as it expects a row ? Or just Buttons plural is enough to make it clear?)